go.mod

```diff
@@ -1,64 +1,65 @@
 module github.com/gardener/gardener-metrics-exporter
 
-go 1.19
+go 1.21
+
+toolchain go1.21.0
 
 require (
-	github.com/gardener/gardener v1.60.0
-	github.com/prometheus/client_golang v1.14.0
-	github.com/sirupsen/logrus v1.9.0
-	github.com/spf13/cobra v1.6.1
-	k8s.io/apimachinery v0.25.4
-	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
+	github.com/gardener/gardener v1.87.2
```

Comment:
Now that the new gardener version is released, maybe we could run:

```
go get -u ./...
go mod tidy
```

again so that at the time the PR is merged, these operations should not yield any changes.

(The PR description would need to be adjusted as well.)